cmd/hz/protobuf/plugin.go

```diff
@@ -364,6 +364,10 @@ func (plugin *Plugin) GenerateFile(gen *protogen.Plugin, f *protogen.File) error
 	if strings.HasPrefix(impt, plugin.Package) {
 		impt = impt[len(plugin.Package):]
 	}
+	// The file name must be the relative path, reference https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/compiler/plugin.proto#L122
+	if strings.HasPrefix(impt, "/") {
+		impt = impt[1:]
```

Comment:
Removing the prefix “/” directly is not always right way to transfer absolute path to relative path?
An implicit premise is that the current work dir needs to be in "/"?

If there is something I missed please let me know🙏